src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$width-height-cell-day: 100px;
```

Comment:
in my opinion, it's bad practice group in one variable width and height

Reply:
Im agree, coz it is more flexible to use parameters for width and height separately. Fixed